english/forest/shelter/play-dead/get-out/get-out.md

```diff
@@ -6,13 +6,12 @@ You make a few steps further, when suddenly a lightning strikes your little shel
 
 Shreds of wood and plants are blasted away and your parachute blazes in flames. 
 You would have surely died if you had stayed in there.
-You feel an unexpected pain: a piece of wood pierced your left leg, causing a deep wound that keeps bleeding.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.